src/api/index.ts

```diff
@@ -16,16 +16,34 @@ import {
   erc20Allowances,
   FEE_TOKEN,
   exchangeOrders,
+  unregisteredTokens,
+  TOKEN_8,
 } from '../../test/data'
 import Web3 from 'web3'
 import { INITIAL_INFURA_ENDPOINT } from 'const'
 
+const isWeb3Mock = process.env.MOCK_WEB3 === 'true'
 const isWalletMock = process.env.MOCK_WALLET === 'true'
 const isTokenListMock = process.env.MOCK_TOKEN_LIST === 'true'
 const isErc20Mock = process.env.MOCK_ERC20 === 'true'
 const isDepositMock = process.env.MOCK_DEPOSIT === 'true'
 const isExchangeMock = process.env.MOCK_EXCHANGE === 'true'
 
+// TODO connect to mainnet if we need AUTOCONNECT at all
```

Comment:
Why this change is in this PR?

Reply:
It's not. It was tracking another branch.